<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainContainer">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="OverlayImageWidget" name="camera1View" native="true">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="RightCams">
        <item>
         <widget class="OverlayImageWidget" name="camera3View" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="OverlayImageWidget" name="camera2View" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIndiConnected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/sync_disabled.svg</normaloff>
     <normalon>:/icons/sync.svg</normalon>:/icons/sync_disabled.svg</iconset>
   </property>
   <property name="text">
    <string>actionIndiConnected</string>
   </property>
  </action>
  <action name="actionMountUnparked">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/gps_not_fixed.svg</normaloff>
     <normalon>:/icons/gps_off.svg</normalon>:/icons/gps_not_fixed.svg</iconset>
   </property>
   <property name="text">
    <string>actionMountUnparked</string>
   </property>
  </action>
  <action name="actionTrackingOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/block.svg</normaloff>
     <normalon>:/icons/target.svg</normalon>:/icons/block.svg</iconset>
   </property>
   <property name="text">
    <string>actionTrackingOn</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OverlayImageWidget</class>
   <extends>QWidget</extends>
   <header>smarttscope.widgets.overlay_image_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
